conf/etcd.conf.yml.example

```diff
@@ -0,0 +1,138 @@
+# This is the configuration file for the etcd server.
+
+# Human-readable name for this member.
+name: %%etcd_name%%
```

Comment:
Could we switch this to use template toolkit now before it hits an official release.

That will make it less of a pain when we put everything with TT in the long run